client/test/fabric/FabricCertificateAuthority.test.ts

```diff
@@ -43,46 +40,13 @@ describe('FabricCertificateAuthority', () => {
     describe('enroll', () => {
 
         it('should return certificate and key from enrollment using JSON connection profile', async () => {
```

Comment:
Should probably change the name of this test to be more accurate now that the code has changed.
E.g. `should return certificate and key from enrollment`

Reply:
Done